{
  "graphName":"ThrowException",
  "resourceType":"OBJECT",
  "startNode":"node1",
  "defaultErrorNode":"node2",
  "nodes":[
    {
      "nodeName":"node1",
      "activityClass":"org.stream.core.test.base.ExceptionThrownActivity"
    },
    {
      "nodeName":"node2",
      "activityClass":"org.stream.core.test.base.TestActivity"
    }
  ]
}
